force-app/main/default/flows/CloneFlow.flow-meta.xml

```diff
@@ -0,0 +1,326 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flow xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
I'm getting an error on deployment:

    Error CloneFlow ToClonedObject (Action) - We can't find an action with the name and action type that you specified.

Can you remove the Flow from pull request and keep only changes in PIN field?